### ISSUE TITLE
text.make() error causing in datatable

### ISSUE THREAD

**Issue by bloggerraviteja** [iampawan/VelocityX#49]
Hi @iampawan 

When I use key.text.make() it throws an error.
When I use default Text noproblem

```
class TransactionHistoryWidget extends StatelessWidget {

  TransactionHistoryWidget({
    @required this.transactions
  }): assert(transactions != null);

  final List<Map<String, dynamic>> transactions;

  @override
  Widget build(BuildContext context) {
    return SingleChildScrollView(
      scrollDirection: Axis.horizontal,
      child: DataTable(
        columns: getColumns(),
        rows: getRows()
      ),
    );
  }

  List<DataColumn> getColumns(){
    List<DataColumn> columns = [];
    columns.addAll(transactions[0].keys.map((key) => DataColumn(label: key.text.make(), tooltip: key)));
    return columns;
  }

  List<DataRow> getRows(){
    List<DataRow> rows = [];
    for(int i = 0; i < transactions.length; i++) {
      List<DataCell>  cells = [];
      cells.addAll(transactions[i].values.map((value) => DataCell(Text(value.toString()))));
      rows.add(DataRow(cells: cells));
    }
    return rows;
  }
}

```

**Comment by iampawan** [comment]
It's a known issue with the DataTable and some other cupertino widgets. It needs to be fixed in the framework

**Comment by bloggerraviteja** [comment]
@iampawan 
Damn, that error took 3hours of time to figure out what's going on.

**Comment by iampawan** [comment]
I am sorry to not point it out in the docs :(

**Comment by bloggerraviteja** [comment]
Yeah. If the documentation is completed it would be helpful :)

Anyway thanks for the package and reply.

**Comment by iampawan** [comment]
You can now use .IsIntrinsic prop with the text widget to support data table kind of widgets. Fixed in v1.3.1
